helm/LICENSE

```diff
@@ -0,0 +1,21 @@
+The MIT License (MIT)
```

Comment:
We've got permission to use MIT instead of Apache?

Reply:
Uuuuuh, no, that's a mistake, it should be the same license as the root.

Reply:
fixed to apache, had MIT from another template NVIDIA helm chart